scripts/packaging/createupmpackages.ps1

```diff
@@ -83,6 +83,7 @@ $packages = [ordered]@{
 # Beginning of the upm packaging script main section
 # The overall structure of this script is:
 #
+# 0) Ensure necessary documentation files (license, changelog, etc.) are copied to the appropriate location

```

Comment:
Nit: If we want to avoid numbering things, it would probably be easier to replace the numbers with bullet points so that we don't have to do highway exit ordering in the future